doc/rtd/explanation/format.rst

```diff
@@ -34,7 +34,7 @@ using a MIME archive.
 
 .. note::
    Cloud config data can also render cloud instance metadata variables using
-   jinja templating. See :ref:`instance_metadata` for more information.
+   ref:`jinja templates <instancedata-Using>`.
```

Comment:
I still think that these notes under each section [aren't the best way to present meta-formats for user-data](https://github.com/canonical/cloud-init/issues/4739 ). However, until that issue gets implemented incremental improvements are welcome, +1.